packages/toolbars-ui/web/src/buttons/SettingsButton/SettingsButton.tsx

```diff
@@ -12,14 +13,14 @@ interface Props {
 const SettingsButton: FC<Props> = ({ toolbarItem }) => {
   const modalService = useContext(ModalContext) || {};
   const { isMobile, t } = useContext(RicosContext) || {};
-
+  const uploadContext = useContext(UploadServiceContext);
   const onClick = toolbarItem.commands.click;
   const node = toolbarItem.attributes.selectedNode;
 
   return (
     <ToggleButton
       Icon={SettingsIcon}
-      onClick={() => onClick({ modalService, isMobile, node })}
+      onClick={() => onClick({ modalService, isMobile, node, uploadContext })}
```

Comment:
```suggestion
      onClick={() => onClick({ modalService, isMobile, node, uploadService })}
```

Please send only uploadService instead of uploadContext